queries.sql

```diff
@@ -36,24 +36,33 @@ SELECT * FROM albums WHERE title LIKE 'A%' ORDER BY title;
 -- Remember: run "\d+ invoices" to see the structure of the "invoices" table
 
 -- Every invoice
+SELECT * FROM invoices; -- these seems to be pretty cut and dry
 
 -- Every invoice ordered by total invoice amount ("total")
+SELECT * FROM invoices ORDER BY total; -- i was thinking either 'total' or 'amount'
 
 -- Every invoice with a total greater than 10
+SELECT * FROM invoices LIKE '>10%'; -- not really sure how to do this one
```

Comment:
Don't guess! Run the query and see if it works as expected.

Try googling "postgres greater than" and see what you get.